pkg/proto/datadog/remoteconfig/remoteconfig.proto

```diff
@@ -100,6 +100,8 @@ message ClientTracer {
   string env = 5;
   string app_version = 6;
   repeated string tags = 7;
+  repeated string process_tags = 9;
```

Comment:
we'll need a PR to add these fields in rc-delivery and currently just merge them to maintain the current behaviour